jekyll/_includes/snippets/built-in-env-vars.adoc

```diff
@@ -0,0 +1,115 @@
+
+| `CIRCLE_PR_NUMBER`             
+| Integer 
+| The number of the associated GitHub or Bitbucket pull request. Only available on forked PRs.
+
+| `CIRCLE_PR_REPONAME`           
+| String  
+| The name of the GitHub or Bitbucket repository where the pull request was created. Only available on forked PRs.
+
+| `CIRCLE_PR_USERNAME`           
+| String  
+| The GitHub or Bitbucket username of the user who created the pull request. Only available on forked PRs.
+
+| `CIRCLE_PREVIOUS_BUILD_NUM`    
+| Integer 
+| The number of previous builds on the current branch.
+
+| `CIRCLE_PROJECT_REPONAME`      
+| String  
+| The name of the repository of the current project.
+
+| `CIRCLE_PROJECT_USERNAME`      
+| String  
+| The GitHub or Bitbucket username of the current project.
+
+| `CIRCLE_PULL_REQUEST`          
+| String  
+| The URL of the associated pull request. If there are multiple associated pull requests, one URL is randomly chosen.
+
+| `CIRCLE_PULL_REQUESTS`         
+| List    
+| Comma-separated list of URLs of the current build's associated pull requests.
+
+| `CIRCLE_REPOSITORY_URL`        
+| String  
+| The URL of your GitHub or Bitbucket repository.
+
+| `CIRCLE_SHA1`                  
+| String  
+| The SHA1 hash of the last commit of the current build.
+
+| `CIRCLE_TAG`                   
+| String  
+| The name of the git tag, if the current build is tagged. For more information, see the [Git Tag Job Execution]({{site.baseurl}}/2.0/workflows/#executing-workflows-for-a-git-tag).
+
+| `CIRCLE_USERNAME`              
+| String  
+| The GitHub or Bitbucket username of the user who triggered the pipeline.
+
+| `CIRCLE_WORKFLOW_ID`           
+| String  
+| A unique identifier for the workflow instance of the current job. This identifier is the same for every job in a given workflow instance.
+
+| `CIRCLE_WORKFLOW_WORKSPACE_ID` 
+| String  
+| An identifier for the [workspace]({{site.baseurl}}/2.0/glossary/#workspace) of the current job. This identifier is the same for every job in a given workspace.
+
+| `CIRCLE_WORKING_DIRECTORY`     
+| String  
+| The value of the `working_directory` key of the current job.
+
+| `CIRCLE_INTERNAL_TASK_DATA`    
+| String  
+| **Internal**. A directory where internal data related to the job is stored. We do not document the contents of this directory; the data schema is subject to change.
+
+| `CIRCLE_COMPARE_URL`           
+| String  
+| **Deprecated**. The GitHub or Bitbucket URL to compare commits of a build. Available in config v2 and below. For v2.1 we will introduce ["pipeline values"]({{site.baseurl}}/2.0/pipeline-variables/) as an alternative.
+
+| `CI_PULL_REQUEST`             
+| String  
+| **Deprecated**. Kept for backward compatibility with CircleCI 1.0. Use `CIRCLE_PULL_REQUEST` instead.
+
+| `CI_PULL_REQUESTS`             
+| List    
+| **Deprecated**. Kept for backward compatibility with CircleCI 1.0. Use `CIRCLE_PULL_REQUESTS` instead.
+|===
```

Comment:
maybe we can remove these now?

Reply:
I would say yes, remove them? Not very confident in my opinion, but I do agree.

Reply:
I'm going to do it! 🗑️